CHANGELOG.md

```diff
@@ -1,10 +1,75 @@
-## vUNRELEASED
+## v1.4.0
 
 #### New features and UX changes
 
-- Add `--user`/`--group` option to rkt run/prepare ([#2419](https://github.com/coreos/rkt/pull/2419)). This option allows overriding the user/group specified in the image manifest.
+- config: add config subcommand ([#2405](https://github.com/coreos/rkt/pull/2405)). This new subcommand prints the current rkt configuration. It can be used to get i.e. authentication credentials.
 
-- Ensure that the initial name and labels used for discovery match the name and labels in the Image Manifest as specified in the appc spec ([#2311](https://github.com/coreos/rkt/pull/2311)). Users wanting the latest image should use `rkt prepare/run/fetch example.com/aci` without any labels. If the discovery server supports the "latest" pattern, the user can bypass a locally cached image in the store and fetch an updated image using `rkt prepare/run/fetch --no-store example.com/aci` option.
+- run: add user/group app flags ([#2419](https://github.com/coreos/rkt/pull/2419)). This adds the `--user`/`--group` options to rkt run/prepare allowing to override the user/group specified in the image manifest.
+
+- gc: Add flag 'mark-only' to mark garbage pods without deleting them ([#2400](https://github.com/coreos/rkt/pull/2400), [2402](https://github.com/coreos/rkt/pull/2402)). This new flag moves exited/aborted pods to the exited-garbage/garbage directory but does not delete them. A third party application can use `rkt gc --mark-only=true` to mark exited pods as garbage without deleting them.
+
+- kvm: Add support for capabilities limitation ([#2222](https://github.com/coreos/rkt/pull/2222)). By default kvm flavor has got enabled every capability inside pod. This patch adds support for a restricted set of capabilities inside a rkt-kvm pod.
+
+- kvm: new volumes support ([#2328](https://github.com/coreos/rkt/pull/2328)). This provides new volumes support based on bind mounting volumes in time of rootfs preparation for the kvm flavor.
+
+- stage1/init: return exit code 1 on error ([#2383](https://github.com/coreos/rkt/pull/2383)). On error, stage1/init was returning a non-zero value between 1 and 7. This change makes it return status code 1 only.
+
+- api: Add 'CreatedAt', 'StartedAt' in pod's info returned by api service. ([#2377](https://github.com/coreos/rkt/pull/2377)).
+
+#### Improved documentation
+
+- documentation: remove superfluous dash ([#2413](https://github.com/coreos/rkt/pull/2413)).
```

Comment:
Usually we're not that detailed with small changes. We just say doc improvements or something like that.